orttraining/orttraining/training_api/include/utils.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
```

Comment:
From my understanding, the files in `include` should be public facing (needed by the consumer of our APIs). Are the functions defined in this file needed to be exposed to users?

Reply:
The utils.h is not intended to be public facing. It is not included in the headers for module and optimizer.